brunch-config.js

```diff
@@ -31,8 +31,7 @@ exports.config = {
   plugins: {
     babel: {
       // Do not use ES6 compiler in vendor code
-      ignore: [/^(web\/static\/vendor)/],
-      loose: "all"
```

Comment:
What are we gaining from upgrading to babel 6? If there is no clear benefit maybe we should wait with the update until there are benefits.